interface-definitions/policy.xml.in

```diff
@@ -1446,9 +1446,14 @@
                         <format>u32:0-4294967295</format>
                         <description>Metric value</description>
                       </valueHelp>
+                      <valueHelp>
+                        <format>&lt;rtt&gt;</format>
```

Comment:
FRR lists the following types:

```
LR1.wue3(config-route-map)# set metric
  (-4294967295-4294967295)  Metric value (use +/- for additions or subtractions)
  +rtt                      Add round trip time
  -rtt                      Subtract round trip time
  rtt                       Assign round trip time
```

Please add all three RTT valueHelps!

Reply:
```
vyos@vyos# set policy route-map TEST rule 10 set metric
Possible completions:
   <+/-metric>          Add or subtract metric
   <0-4294967295>       Metric value
   <+/-rtt>             Add or subtract round trip time
   <rtt>                Round trip time

```